test/getTestName.js

```diff
@@ -0,0 +1,13 @@
+const { CI_JOB_ID, CI_PIPELINE_ID, CI_COMMIT_SHORT_SHA } = process.env
+
+function getCIInfos() {
+  if (CI_PIPELINE_ID && CI_JOB_ID && CI_COMMIT_SHORT_SHA) {
+    return `job: ${CI_JOB_ID} commit: ${CI_COMMIT_SHORT_SHA}`
+  }
```

Comment:
Could it be useful to have `${process.env.USER} (local)` for local runs?

Reply:
good idea!